package.json

```diff
@@ -66,7 +66,8 @@
     "typedoc-plugin-markdown": "^3.16.0",
     "typescript": "^4.3.5",
     "web-vitals": "^1.0.1",
-    "yarn": "^1.22.17"
+    "yarn": "^1.22.17",
+    "zod": "^3.22.4"
```

Comment:
Why this package is added ?

Reply:
@noman2002 [Zod](https://zod.dev/) is used for validation of input given by the user.

Reply:
But we already have validation in various places without any package and its a very basic thing. I think it is not required to use any package for validation.

@aashimawadhwa your thoughts on this.

Reply:
yes i think we can make validation work without the zod package too. @MahendraDani @noman2002

Reply:
> yes i think we can make validation work without the zod package too. @MahendraDani @noman2002

@aashimawadhwa @noman2002 
I guess you are right. Should I close this PR?

Reply:
@aashimawadhwa @noman2002 
My aim was to validate input for special characters and other characters like `[`, `]`, `{`, `}`,`@`, `#` etc. 
But it seems that when we validate any input by string `zod` considers the above characters in string too so using `zod` does not help that much.

Reply:
@MahendraDani why are you thinking on closing this PR ?, you can just remove zod and use TS or Regex to validate the strings

Reply:
Will it be fine if I use regex and remove the package `zod` ?

Reply:
Yeah sure go ahead !